pkgs/development/libraries/boost/generic.nix

```
@@ -113,6 +114,14 @@ stdenv.mkDerivation {
     if version == "1.55.0"
     then ./darwin-1.55-no-system-python.patch
     else ./darwin-no-system-python.patch)
+  # Fix boost-context segmentation faults on ppc64 due to ABI violation
+  ++ optional (versionAtLeast version "1.61" &&
+               versionOlder version "1.71") (fetchpatch {
```

```suggestion
  ++ optional (
    versionAtLeast version "1.61"
    && versionOlder version "1.71"
    && stdenv.hostPlatform.isPower
  ) (fetchpatch {
```

@SuperSandro2000 Should be as simple as adding a `stdenv.hostPlatform.isPower` check …

---

That's what I had originally, but then I saw this rationale https://github.com/NixOS/nixpkgs/pull/100807#discussion_r508640789. Although I suppose there's no risk of this patch rotting since it only applies to older versions